### DISCUSSION TITLE
file permission problem

### DISCUSSION THREAD

**Discussion by octalgenesis** [transmission/transmission#7334] [Support]
i have files on my android phone that i have downloaded and then removed the torrent link, now the files are in some sort of permission vaccum cause i can see the folders when i try to change the download location but i cant browse it with a file manager, it only gives me a error. Since your GUI doesnt allow files to be moved or deleted they are now stuck in limbo and i wonder how to solve this issue. I am afraid that the files will be stuck on the internal storage forever and just waste space. i thought of deleting the app itself in hope of it deleteing the files but its not part of the cache size either. Which is why i am turning to you for some sort of guidence on the matter. If my issue doesnt make sense just ask more questions i will try to provide more info thanks

**Comment by tearfur** [comment]
Whatever app you downloaded, it's not made by us. We don't have an official Android release.
